pkg/framework/context/context.go

```diff
@@ -28,6 +29,7 @@ type Provider struct {
 	storeProvider          storage.Provider
 	transientStoreProvider storage.Provider
 	kms                    legacykms.KMS
+	secretLock             secretlock.Service
```

Comment:
Q: why isn't the secret lock injected into the KMS?

Reply:
the new kms is not created yet